walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/ListTasksServlet.java

```diff
@@ -34,26 +34,25 @@
 @WebServlet("/list-tasks")
 public class ListTasksServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
-
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    PreparedQuery results = datastore.prepare(query);
-
-    List<Task> tasks = new ArrayList<>();
-    for (Entity entity : results.asIterable()) {
-      long id = entity.getKey().getId();
-      String title = (String) entity.getProperty("title");
-      long timestamp = (long) entity.getProperty("timestamp");
-
-      Task task = new Task(id, title, timestamp);
-      tasks.add(task);
+    @Override
```

Comment:
i think diffs are still picking up changes to this file because the indentation changed. please fix indentation